lib/chef/knife/zero_converge.rb

```diff
@@ -20,6 +20,11 @@ class ZeroConverge < Chef::Knife::BootstrapSsh
       self.options = Ssh.options.merge(self.options)
       self.options[:use_sudo_password] = Bootstrap.options[:use_sudo_password]
 
+      # fatal by default Ref: https://github.com/chef/chef/pull/6781
+      if self.options[:duplicated_fqdns]
```

Comment:
__[RuboCop]__

Style/IfUnlessModifier: Favor modifier `if` usage when having a single-line body. Another good alternative is the usage of control flow `&&`/`||`.


<!-- Comment by [SideCI](https://sideci.com) -->